test/built-ins/Temporal/PlainDateTime/comparison.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Another naming nitpick, sorry ... "comparison" sounds like it should be testing the compare() method. I'd actually suggest moving this one into the `built-ins/Temporal/PlainDateTime/prototype/valueOf` directory because that is really what it's testing. Check `built-ins/Temporal/PlainDate/prototype/valueOf/basic.js` for reference (although your way is more explicit with the assertion messages, which is good.)

Reply:
sure, sounds good -- done